<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>901</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="dfRowsScrollDownbutton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Scroll Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dfRowsScrollUpbutton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Scroll Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dfRowsTopbutton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dfRowsFindNextbutton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dfRowsHelpbutton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>178</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="dfRowsSplashScreen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>1171</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="dfRowsLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="dfRowsBrowsedfbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse Selected df</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
